nixos/modules/services/networking/opkssh/opkssh.nix

```diff
@@ -0,0 +1,171 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
```

Comment:
We tend to use `lib` directly where it's needed for visibility, `with lib;` at the top of the file makes it hard to identify where things come from.